go-console/.vscode/tasks.json

```diff
@@ -0,0 +1,919 @@
+      "command": "sshpass",
+      "type": "process",
+      "args": [
+        "-p",
+        "${config:torizon_psswd}",
+        "ssh",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "torizon@${config:torizon_ip}",
+        "LOCAL_REGISTRY=${config:host_ip} TAG=arm docker-compose pull __container__-debug"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "run-container-torizon-debug-arm",
+      "detail": "hide",
+      "command": "sshpass",
+      "type": "process",
+      "args": [
+        "-p",
+        "${config:torizon_psswd}",
+        "ssh",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "torizon@${config:torizon_ip}",
+        "LOCAL_REGISTRY=${config:host_ip} TAG=arm docker-compose up -d __container__-debug"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "deploy-torizon-arm",
+      "detail": "hide",
+      "command": "scp",
+      "type": "process",
+      "args": [
+        "-i",
+        "${workspaceFolder}/.conf/id_rsa",
+        "-o",
+        "UserKnownHostsFile=/dev/null",
+        "-o",
+        "StrictHostKeyChecking=no",
+        "-P",
+        "${config:torizon_debug_ssh_port}",
+        "-pr",
+        "${workspaceFolder}/target/armv7-unknown-linux-gnueabihf/debug",
+        "torizon@${config:torizon_ip}:~/app"
+      ],
+      "dependsOn": [
+        "copy-docker-compose",
+        "pre-cleanup-arm",
+        "build-debug-arm",
+        "build-container-torizon-debug-arm",
+        "push-container-torizon-debug-arm",
+        "pull-container-torizon-debug-arm",
+        "run-container-torizon-debug-arm"
+      ],
+      "dependsOrder": "sequence",
+      "problemMatcher": "$msCompile",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
+      "label": "wait-debugpy-start-arm",
+      "detail": "hide",
+      "command": "sleep",
+      "type": "process",
+      "args": ["2"],
+      "dependsOn": [
+        "validate-settings",
+        "validate-arch-arm",
+        "apply-torizon-packages",
+        "copy-docker-compose",
+        "pre-cleanup-arm",
+        "build-debug-arm",
+        "build-container-torizon-debug-arm",
+        "push-container-torizon-debug-arm",
+        "pull-container-torizon-debug-arm",
+        "run-container-torizon-debug-arm",
+        "wait-a-bit",
+        "pos-cleanup",
+        "deploy-torizon-arm",
+        "start-torizon-debug-arm"
+      ],
+      "dependsOrder": "sequence",
+      "icon": {
+        "id": "flame",
+        "color": "terminal.ansiYellow"
+      }
+    },
+    {
```

Comment:
I believe all the functions here all the way to the end of the `validate-arch-riscv64` function (ends at line 831) can be removed from here as they get added automatically by the extension when the project is created. Their implementation is common to all projects here: [https://github.com/toradex/vscode-torizon-templates/blob/main/assets/tasks/common.json](https://github.com/toradex/vscode-torizon-templates/blob/main/assets/tasks/common.json).

Please correct this comment if there is something wrong on it @microhobby